src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -166,6 +165,8 @@ MenuItemList NotationContextMenuModel::makeElementItems()
 
     if (isSingleSelection()) {
         items << makeMenu(qtrc("notation", "Select"), makeSelectItems());
+    } else if (canSelectSimilarInRange()) {
+        items << makeMenu(qtrc("notation", "Select"), MenuItemList { makeMenuItem("select-similar-range"), makeMenuItem("select-dialog") });
     }
```

Comment:
this might not be quite right actually, the select-dialog should probably be available even if there's a list selection (multiple individual elements selected) and you right-click on an element?